optee-teec/optee-teec-sys/build.rs

```diff
@@ -43,7 +43,14 @@ fn link() {
 
     let optee_client_dir =
         env::var(ENV_OPTEE_CLIENT_EXPORT).expect("OPTEE_CLIENT_EXPORT is not set");
-    let search_path = Path::new(&optee_client_dir).join("usr/lib");
-    println!("cargo:rustc-link-search={}", search_path.display());
+    let library_path = PathBuf::from(optee_client_dir).join("usr/lib");
+    if !library_path.exists() {
+        panic!(
+            "Library path {} does not exist",
```

Comment:
"OPTEE_CLIENT_EXPORT usr/lib path {} does not exist"

Reply:
Try this build cmd:
```
(cd $OPTEE_DIR/optee_client && make WITH_TEEACL=0 DESTDIR=$PWD/export_arm32 CROSS_COMPILE=$CROSS_COMPILE32)
(cd $OPTEE_DIR/optee_client && make clean && make WITH_TEEACL=0 DESTDIR=$PWD/export_arm64 CROSS_COMPILE=$CROSS_COMPILE64)
```
ref: https://github.com/apache/teaclave-trustzone-sdk/blob/main/build_optee_libraries.sh#L50-L51

You can run `build_optee_libraries.sh` for quickly build OP-TEE libraries.